icubam/www/handlers/home.py

```diff
@@ -40,6 +53,16 @@ def initialize(self, config, db_factory):
     super().initialize(config, db_factory)
     self.regions = None
 
+  def get_disclaimer_url(self):
+    if self.config.server.has_key('disclaimer'):
+      path = self.config.server.disclaimer
+      # The user has already agreed, we skip.
+      if path is not None and os.path.exists(path):
```

Comment:
Can you actually put `None` as a value in toml without it being interpreted as a string? Just curious.

Reply:
Bad copy/paste comment. Fixed on 2769ea7

Reply:
You are right. If I set a value to `None` in the toml file, I am facing a `TomlDecodeError`. Removed this test on b2ad56e